src/main/resources/config/liquibase/master.xml

```diff
@@ -118,6 +118,8 @@
     <include file="classpath:config/liquibase/changelog/20210413105400_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20210417123460_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20210507092500_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20210515191046_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20210519162242_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20210528104100_changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
(Check that we can leave the order like that)

Reply:
Fixed in https://github.com/ls1intum/Artemis/pull/3457/commits/db354dd219003f0c2202f3a4478708e37b7bf533 together with some other requests in this file

Reply:
Had a look a `changelog/20210528104100_changelog.xml` and it does no interfere with our changes, also tested executing changelog/20210528104100_changelog.xml before the ones that were added in this PR and we didn't note any unexpected behaviour